starter-code/scripts/controllers/articleController.js

```diff
@@ -4,7 +4,13 @@
   var articleController = {};
 
   articleController.reveal = function() {
-    /* TODO: Use your DOM skills to reveal only the articles section! */
+    /* TODO: DONE! Use your DOM skills to reveal only the articles section! */
+    $('.main-nav').on('click', '.tab', function(e) {
+      e.preventDefault();
+      $('.tab-content').hide();
+      $('#' + $(this).data('content')).fadeIn();
+    });
+    $('.main-nav .tab:first').click();
```

Comment:
Same comments here as above! No need to use event listeners; simply hide and show the appropriate sections.